src/platform/common/platform/fileSystem.web.ts

```diff
@@ -0,0 +1,113 @@
+import * as vscode from 'vscode';
```

Comment:
I don't think this file is necessary. The common FileSystem is identical to this.

Reply:
Meaning I don't think you need a web FileSystem. The common FileSystem should already work.

Reply:
One difference is that in Node, `fsPath` and `Uri.file` were used, but they don't work in web. Am I misinterpreting that?

Reply:
The common `FileSystem` is meant to use `vscode.workspace.fs` for all the operations, and the `FileSystem.node` override some if `nodejs` fs is required. The methods in this file are safe to move into the common `FileSystem`.